doc/classes/CollisionPolygon.xml

```diff
@@ -1,8 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="CollisionPolygon" inherits="Spatial" category="Core" version="3.0.alpha.custom_build">
 	<brief_description>
+		Editor-only class for defining a collision polygon in 3D space.
 	</brief_description>
 	<description>
+		Edits a collision polygon's verticies on a plane and give it depth perpendicular to that plane. Editor-only class. Creates a [Shape] for gameplay. Properties modified during gameplay will have no effect.
```

Comment:
"Allows editing" sounds better - the node doesn't edit the vertices, you do.
verticies -> vertices

"on a plane and give it depth" has a tense problem.  Try:
Allows editing a collision polygon's vertices on a selected plane. Can also set a depth perpendicular to that plane.

Other comments:
This class is only available in the editor.  It will not appear in the scene tree at runtime.